### DISCUSSION TITLE
Enable Brave contribution for this awesome project!

### DISCUSSION THREAD

**Discussion by Coexo** [upscayl/upscayl#773] [Ideas]
![Screenshot 2024-04-14 174522](https://github.com/upscayl/upscayl/assets/77916887/70244f4d-a6e8-4206-8fe6-b171be81e4c8)

If the dev can sign in for Brave Creators program. Users using brave browser can send contributions for this project through https://www.upscayl.org/ and https://github.com/upscayl.

**Comment by NayamAmarshe** [comment]
Hiya! I am a Brave user too but I stopped using Rewards because they're not available in my country 😅
